### PR TITLE
[reorg] New sidebar!

### LINKED ISSUE #8433: There are many ways to create a blog with Astro. How to choose one?
### 📋 Explain your issue

Right now, there are many ways that one could create a blog with Astro. It would be nice if the docs listed the ways, and gave the reader some guidance on how to choose one. (It probably depends on what the reader wants to do. They need guidance.)

As a starting point, how are the following things related?

- There is a "Build a Blog" tutorial in the docs. It builds this: https://github.com/withastro/blog-tutorial-demo
- There is the "Astro Starter Kit: Blog". See https://github.com/withastro/templates/tree/main/templates/blog 
- There are many Astro themes for blogging, both free and paid: https://astro.build/themes/?search=&categories%5B%5D=blog 
- One can use "Content Collections", i.e. https://docs.astro.build/en/guides/content-collections/ 
- There are Astro integrations with external CMSes, see: https://docs.astro.build/en/guides/cms/ 
- There's "Astro DB", "Astro Studio" and "Astro Studio CMS".

### PR DESCRIPTION
**Pull request by sarah11918**
This PR attempts to tame our wild sidebar!

## The goals of this PR:

- a more helpful organization of the sidebar nav. As it has grown and we have added more and more features, it has become more difficult to use the sidebar to quickly find what you need.
- a nicer "new user journey" for those entering the docs for the first time. 

### Non-goals of *this* PR right now (although they could be future goals!)
- Significant content updates. We generally are not making individual pages better, just the sidebar.
- Reintroducing our old Learn/Reference split from before Starlight.

### Changing from headings emphasizing content *type* to headings describing *topics included*.
- "Guides" is maybe no longer as helpful as it once was now that we have so many of them, and it's one big pile of them.
- We had started to separate some things out to draw more attention to them as key features, but this only made more sections that were "categories" (e.g. Built in Features) that didn't give any indication as to what would be included.
- These new headings attempt to categorize a bunch of our pages by topics, so that even if collapsed (but we won't!), you would still have helpful ideas of where the page you wanted exists
- Note that it is *still* worth keeping some original headings because the *type* of page itself is helpful for readers wanting that kind of information:
	- Basics/Concepts: things you probably should know/understand before you try to go too far. (It's helpful for readers to get that idea)
	- Reference: the pages where you'll want to look up stuff without much fluff. (It's helpful for readers to know where the most technical details are for quickly checking things)

### Rethinking the "new user journey".

#8415 

- Our current "install, run, good luck!" page is short, which admittedly looks not scary, but not as *helpful* as it could be. 
- We often find that some things from the tutorial, which are more intro-friendly and hand-holding, are things we point new users to, but pointing them somewhere mid-way through a tutorial doesn't make much sense
- We want to set new users up for success, so even though I've made the "install" page longer, I think the extra content gives new users a sense of support, and where to go.
- Specifically, our advice on starting from a new theme/template has always been a sub-par experience because it comes AFTER starting a new project... at which point it's too late to "add a theme".

### Eliminating *some* things from the sidebar nav entirely
- Everything that has been removed is *still* discoverable, perhaps from a page with more context. (And, CTRL+F works!)
- Fewer items in the sidebar lets us control the reader journey more to higher level intro spots, and the pages themselves do the work of linking to more details.
- Specifically, 
	- the section on **configuring Astro** has been removed *from the sidebar* (where it was unceremoniously located near the bottom anyway) and is now highlighted right on the "Install and Setup" page as "Next Steps" with fancy cards. This should ensure that things like setting up the editor, configuring TypeScript and import aliases, which are often "early and once" actions, are visible and promoted when someone is starting a new project!
	- Connect to Astro Studio has been included in the Astro DB guide page #8414 

## Tasks to complete

- [x] All appropriate redirects for files that actually changed path (`/recipes/` -> `en/community-resources/content/#official-recipes` and `install/auto` / `install/manual` -> `installation-and-setup` and `aliases` -> `imports`
- [x] Check consistent casing for all section headings/page entries 
- [x] `isSubPage` logic so the right page in the sidebar nav is highlighted for non-entries
	- Tutorial highlighted for Extensions (Extend with CC/VT)
	- Installation and Setup highlighted for: Editor Setup, Dev Toolbar
- [ ] Lunaria checks
- [x] Move to "old translations": `install/auto` , `install/manual`, and `recipes` in non-EN languages so that translators can use the content

## PRs that must merge first

- #8414 
- #8415 
- #8507 
- #8508 
- #8511 
- #8515 
- #8513 

## Closes Issues

- closes #8433 by exposing the currently "hidden" Authoring Content page

### CODE REVIEW

File: src/i18n/en/nav.ts [hunk 1]
```diff
@@ -18,190 +18,105 @@
...
-		key: 'reference/integrations-reference',
-	},
+	{ text: 'Configuration', slug: 'reference/configuration-reference', key: 'reference/configuration-reference' },
+	{ text: 'Astro Runtime API', slug: 'reference/api-reference', key: 'reference/api-reference' },
+	{ text: 'Astro CLI', slug: 'reference/cli-reference', key: 'reference/cli-reference' },
+	{ text: 'Directives Reference', slug: 'reference/directives-reference', key: 'reference/directives-reference' },
+	{ text: 'Error Reference', slug: 'reference/error-reference', key: 'reference/error-reference' },
+
+
+	{ text: 'Other Develoment APIs', header: true, type: 'api', key: 'dev' },
```
Comment (VoxelMC) [inline src/i18n/en/nav.ts:94]:
```suggestion
	{ text: 'Other Astro APIs', header: true, type: 'api', key: 'dev' },
```
Note: Typo here if we keep it as "Development".

I think it would help to make it obvious that this is a section for developer tools.
Perhaps "Astro" APIs, is not correct, though. Maybe "Extend Astro" or "Create an Integration" could also work.

File: src/content/docs/en/install/auto.mdx [hunk 2]
```diff
@@ -1,154 +1,414 @@
...
+      # create a new project with yarn
+      yarn create astro
+      ```
+      </TabItem>
+    </Tabs>
+
+    You can run `create astro` anywhere on your machine, so there's no need to create a new empty directory for your project before you begin. If you don't have an empty directory yet for your new project, the wizard will help create one for you automatically.
 
-If you aren't able to open your project in the browser, go back to the terminal where you ran the `dev` command and look to see if an error occurred, or if your project is being served at a different URL than the one linked to above.
+    If all goes well, you should see a success message followed by some recommended next steps. `cd` into your new project directory to begin using Astro.
```
Comment (VoxelMC) [inline src/content/docs/en/install/auto.mdx:56]:
```suggestion
    If all goes well, you will see a success message followed by some recommended next steps. `cd` into your new project directory to begin using Astro.
```
Perhaps a more hopeful option!

File: src/content/docs/en/install/auto.mdx
Comment (VoxelMC) [inline src/content/docs/en/install/auto.mdx:1]: I really love the way you have laid this out. It is quite encompassing yet concise!

File: src/content/docs/en/install/auto.mdx [hunk 3]
```diff
@@ -1,154 +1,414 @@
...
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run build
+      ```
+      </TabItem>
+    </Tabs>
+
+Astro will build a deploy-ready version of your site  in a separate folder (`dist/` by default) and you can watch its progress in the terminal. This will alert you to any build errors in your project before you deploy to production.
```
Comment (VoxelMC) [inline src/content/docs/en/install/auto.mdx:348]:
```suggestion
Astro will build a deploy-ready version of your site in a separate folder (`dist/` by default) and you can watch its progress in the terminal. This will alert you to any build errors in your project before you deploy to production. If you opted to use Typescript, the `build` script will also check your project for type errors.
```
Not sure of the accuracy here, but perhaps important to mention. I remember this being discussed quite a bit ago, and how it is not abundantly clear that the `create` command adds this to your `package.json` when using Typescript.

File: src/content/docs/en/community-resources/content.mdx [hunk 4]
```diff
@@ -37,11 +37,9 @@ Check out the following courses and tutorials to learn more about Astro.
 
 ## Recipes and Guides
 
-See guided examples of adding features to your Astro project, submitted by our community members!
+[Astro's official recipes](/en/recipes/) are short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
```
Comment (VoxelMC) [inline src/content/docs/en/community-resources/content.mdx:40]:
```suggestion
[Astro's official recipes](/en/recipes/) are quick, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
```
I think this would make it sound like the recipes are concise, rather than simple, and perhaps encourage more people to use them. I might ALSO be overthinking!

File: src/i18n/en/nav.ts [hunk 5]
```diff
@@ -18,190 +18,105 @@
...
-	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
-	{ text: 'More recipes', slug: 'recipes', key: 'guides/recipes' },
+	{ text: 'Astro Template Syntax', slug: 'basics/astro-syntax', key: 'basics/astro-syntax' },
+	{ text: 'Rendering Modes', slug: 'basics/rendering-modes', key: 'basics/rendering-modes' },
+	{ text: 'Imports', slug: 'guides/imports', key: 'guides/imports' },
+	{ text: 'Integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
 
-	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
+	
+	{ text: 'Create Routes', header: true, type: 'learn', key: 'routes' },
```
Comment (VoxelMC) [inline src/i18n/en/nav.ts:43]:
```suggestion
	{ text: 'Create routes', header: true, type: 'learn', key: 'routes' },
```

File: src/i18n/en/nav.ts [hunk 6]
```diff
@@ -18,190 +18,105 @@
...
-		text: 'Project Structure',
-		slug: 'basics/project-structure',
-		key: 'basics/project-structure',
-	},
-	{
-		text: 'Components',
-		slug: 'basics/astro-components',
-		key: 'basics/astro-components',
-	},
+	{ text: 'Learn the Basics', header: true, type: 'learn', key: 'basics' },
```
Comment (VoxelMC) [inline src/i18n/en/nav.ts:32]:
```suggestion
	{ text: 'Learn the basics', header: true, type: 'learn', key: 'basics' },
```

File: src/i18n/en/nav.ts [hunk 7]
```diff
@@ -18,190 +18,105 @@
...
 	{ text: 'Getting Started', slug: 'getting-started', key: 'getting-started' },
-	{ text: 'Installation', slug: 'install/auto', key: 'install' },
-	{ text: 'Editor Setup', slug: 'editor-setup', key: 'editor-setup' },
-	{ text: 'Upgrade Astro', slug: 'upgrade-astro', key: 'upgrade-astro' },
+	{ text: 'Installation and Setup', slug: 'install/auto', key: 'install/auto' },
+	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
+	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
+	{ text: 'Introductory Tutorial - Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
 
 	{ text: 'Core Concepts', header: true, type: 'learn', key: 'coreConcepts' },
```
Comment (VoxelMC) [inline src/i18n/en/nav.ts:28]:
```suggestion
	{ text: 'Core concepts', header: true, type: 'learn', key: 'coreConcepts' },
```

File: src/i18n/en/nav.ts [hunk 8]
```diff
@@ -18,190 +18,105 @@
...
+	{ text: 'Content Collections', slug: 'guides/content-collections', key: 'guides/content-collections' },
+	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Add an RSS feed', slug: 'guides/rss', key: 'guides/rss' },
+
+
+
+	{ text: 'Connect your data', header: true, type: 'learn', key: 'data' },
+	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
+	{ text: 'Astro DB', slug: 'guides/astro-db', key: 'guides/astro-db' },
+	{ text: 'Add backend services', slug: 'guides/backend', key: 'guides/backend' },
```
Comment (VoxelMC) [inline src/i18n/en/nav.ts:64]:
```suggestion
	{ text: 'Add Backend Services', slug: 'guides/backend', key: 'guides/backend' },
```

File: src/i18n/en/nav.ts [hunk 9]
```diff
@@ -18,190 +18,105 @@
...
+	{ text: 'Authoring Content', slug: 'guides/content', key: 'guides/content' },
 	{ text: 'Markdown', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
-	{
-		text: 'Scripts & Event Handling',
-		slug: 'guides/client-side-scripts',
-		key: 'guides/client-side-scripts',
-	},
+	{ text: 'Content Collections', slug: 'guides/content-collections', key: 'guides/content-collections' },
+	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Add an RSS feed', slug: 'guides/rss', key: 'guides/rss' },
```
Comment (VoxelMC) [inline src/i18n/en/nav.ts:57]:
```suggestion
	{ text: 'Add an RSS Feed', slug: 'guides/rss', key: 'guides/rss' },
```

**Review by VoxelMC — COMMENTED** [review]
Hello, I am back! This is truly awesome. Thank you for spearheading this. 

I added some comments about wording and suggested the addition of a sentence. I also made some edits for consistency's sake for sidebar title formatting!

Hope it helps!

**Comment by lorenzolewis** [comment]
possibly slightly re-order the upper-level sections to be like this:

- Welcome, World!
- Core Concepts
- Learn the Basics
- Manage your Content
- Assets
- Create Routes
- Connect your Data
- Client Side Interactivity
- …the rest of the entries 

I bumped up “Manage your Content” and “Assets” because I personally need to reference them more. Creating routes I only need IF I’m doing dynamic routing. 

I also noticed that image and asset topics seem to pop up more in support than routing it feels like.

File: src/content/docs/en/install/auto.mdx (hunk 3 above)
Comment (sarah11918) [reply src/content/docs/en/install/auto.mdx:348]: Good call! 💪

File: src/content/docs/en/community-resources/content.mdx (hunk 4 above)
Comment (sarah11918) [reply src/content/docs/en/community-resources/content.mdx:41]: Our official recipes ARE concise! Quick refers to speed, and there's no guarantee how long it takes you to do them. But the ones we have are actually pretty short!

File: src/i18n/en/nav.ts (hunk 6 above)
Comment (sarah11918) [reply src/i18n/en/nav.ts:32]: Our current standard is upper case for section headings in the nav bar, so I will leave all as that for now and then make one sweeping decision.

File: src/i18n/en/nav.ts (hunk 8 above)
Comment (sarah11918) [reply src/i18n/en/nav.ts:64]: In fact, this matches more with "Connect your data" and it might be "Data Fetching" that's out of place. We'll take one sweep for page titles!

**Comment by sarah11918** [comment]
Thank you, @lorenzolewis  - will play with this idea and see how it feels!

**Comment by sarah11918** [comment]
To self: I just moved the `install/auto` and `install/manual` files to "old translations" to let link check confirm that we caught all the English links. When link check passes, I'll move the other language files to see which links still might need to be fixed, too!

File: src/i18n/en/nav.ts [hunk 10]
```diff
@@ -18,194 +18,106 @@
...
+	{ text: 'Connect Your Data', header: true, type: 'learn', key: 'data' },
+	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
+	{ text: 'Astro DB', slug: 'guides/astro-db', key: 'guides/astro-db' },
+	{ text: 'Add Backend Services', slug: 'guides/backend', key: 'guides/backend' },
 	{ text: 'E-commerce', slug: 'guides/ecommerce', key: 'guides/ecommerce' },
-	{ text: 'Testing', slug: 'guides/testing', key: 'guides/testing' },
 	{ text: 'Authentication', slug: 'guides/authentication', key: 'guides/authentication' },
+
+
+	{ text: 'Client-side Interactivity', header: true, type: 'learn', key: 'clientSide' },
```
Comment (lorenzolewis) [inline src/i18n/en/nav.ts:76]:
```suggestion
	{ text: 'Client-Side Interactivity', header: true, type: 'learn', key: 'clientSide' },
```
```suggestion
	{ text: 'Client-side Interactivity', header: true, type: 'learn', key: 'clientSide' },
```

File: src/i18n/en/nav.ts [hunk 11]
```diff
@@ -18,194 +18,106 @@
...
+	{ text: 'UI Frameworks', slug: 'guides/framework-components', key: 'guides/framework-components' },
+	{ text: 'Scripts & Event Handling', slug: 'guides/client-side-scripts', key: 'guides/client-side-scripts' },
+
+
+	{ text: 'Maintain Your project', header: true, type: 'learn', key: 'maintain' },
```
Comment (lorenzolewis) [inline src/i18n/en/nav.ts:81]:
```suggestion
	{ text: 'Maintain Your Project', header: true, type: 'learn', key: 'maintain' },
```
```suggestion
	{ text: 'Maintain Your project', header: true, type: 'learn', key: 'maintain' },
```

File: src/i18n/en/nav.ts [hunk 12]
```diff
@@ -18,194 +18,106 @@
...
-	{ text: 'Start Here', header: true, type: 'learn', key: 'startHere' },
+	{ text: 'Welcome, World!', header: true, type: 'learn', key: 'welcomeWorld' },
 	{ text: 'Getting Started', slug: 'getting-started', key: 'getting-started' },
-	{ text: 'Installation', slug: 'install/auto', key: 'install' },
-	{ text: 'Editor Setup', slug: 'editor-setup', key: 'editor-setup' },
-	{ text: 'Upgrade Astro', slug: 'upgrade-astro', key: 'upgrade-astro' },
+	{ text: 'Installation and Setup', slug: 'install-and-setup', key: 'install-and-setup' },
+	{ text: 'Deploy Your Site', slug: 'guides/deploy', key: 'guides/deploy' },
+	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
+	{ text: 'Introductory Tutorial - Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
```
Comment (lorenzolewis) [inline src/i18n/en/nav.ts:26]:
```suggestion
	{ text: 'Tutorial: Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
```

File: src/i18n/en/nav.ts [hunk 13]
```diff
@@ -18,194 +18,106 @@
...
-	},
-	{ text: 'Middleware', slug: 'guides/middleware', key: 'guides/middleware' },
+
+	{ text: 'Connect Your Data', header: true, type: 'learn', key: 'data' },
+	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
+	{ text: 'Astro DB', slug: 'guides/astro-db', key: 'guides/astro-db' },
+	{ text: 'Add Backend Services', slug: 'guides/backend', key: 'guides/backend' },
 	{ text: 'E-commerce', slug: 'guides/ecommerce', key: 'guides/ecommerce' },
-	{ text: 'Testing', slug: 'guides/testing', key: 'guides/testing' },
 	{ text: 'Authentication', slug: 'guides/authentication', key: 'guides/authentication' },
```
Comment (TheOtterlord) [inline src/i18n/en/nav.ts:73]:
```suggestion
	{ text: 'Authentication', slug: 'guides/authentication', key: 'guides/authentication' },
	{ text: 'Environment Variables', slug: 'guides/environment-variables', key: 'guides/environment-variables' },
```

File: src/util/isSubPage.ts [hunk 14]
```diff
@@ -24,6 +24,16 @@ const typeIndexes: Partial<Record<CollectionEntry<'docs'>['data']['type'], strin
...
 	'Error Reference': 'reference/error-reference',
+	Tutorials: 'tutorial/0-introduction',
+};
+
+/** Map of page slugs to the slug of their parent page. */
+const parentIndex: Record<string, string> = {
+	'editor-setup': 'install-and-setup',
+	'guides/typescript': 'install-and-setup',
+	'guides/aliases': 'install-and-setup',
+	'guides/environment-variables': 'install-and-setup',
```
Comment (TheOtterlord) [inline src/util/isSubPage.ts:35]:
```suggestion
```

**Review by TheOtterlord — COMMENTED** [review]
Adds environment variables to sidebar

File: src/util/isSubPage.ts [hunk 15]
```diff
@@ -24,6 +24,15 @@ const typeIndexes: Partial<Record<CollectionEntry<'docs'>['data']['type'], strin
...
 const categoryIndex: Partial<Record<ReturnType<typeof getPageCategory>, string>> = {
 	'Error Reference': 'reference/error-reference',
+	Tutorials: 'tutorial/0-introduction',
+};
+
+/** Map of page slugs to the slug of their parent page. */
+const parentIndex: Record<string, string> = {
+	'editor-setup': 'install-and-setup',
+	'guides/typescript': 'install-and-setup',
+	'guides/aliases': 'install-and-setup',
```
Comment (sarah11918) [inline src/util/isSubPage.ts:34]:
```suggestion
```

File: vercel.json [hunk 16]
```diff
@@ -5,6 +5,8 @@
   "cleanUrls": true,
 
   "redirects": [
+    { "source": "/en/install/auto/", "destination": "/en/install-and-setup/" },
+    { "source": "/en/install/manual/", "destination": "/en/install-and-setup/" },
```
Comment (delucis) [inline vercel.json:9]:
```suggestion
    { "source": "/:lang/install/auto/", "destination": "/:lang/install-and-setup/" },
    { "source": "/:lang/install/manual/", "destination": "/:lang/install-and-setup/" },
```

File: vercel.json [hunk 17]
```diff
@@ -93,7 +95,9 @@
 
     { "source": "/en/guides/debugging/", "destination": "/en/guides/troubleshooting/" },
 
-    { "source": "/en/quick-start/", "destination": "/en/installation/" },
+    { "source": "/en/quick-start/", "destination": "/en/install-and-setup/" },
+
+    { "source": "/en/installation/", "destination": "/en/install-and-setup/" },
```
Comment (delucis) [inline vercel.json:100]: Why was this added? I don’t see a current `/installation/` link in use

**Review by delucis — COMMENTED** [review]
Spotted a couple of details in the redirects!

File: vercel.json (hunk 17 above)
Comment (sarah11918) [reply vercel.json:100]: This was preexisting, and I thought it better to update to be safe? I'm not sure anyone might still be linking to `quick-start` at this point, but I noticed it pointing to something so thought I should update the link in the chain.

**Comment by sarah11918** [comment]
!coauthor

**Comment by sarah11918** [comment]
Full co-author dump from all earlier PRs:

Co-authored-by: Lorenzo Lewis <15347255+lorenzolewis@users.noreply.github.com>
Co-authored-by: Chris Swithinbank <357379+delucis@users.noreply.github.com>
Co-authored-by: voxel!() <voxelmc@hotmail.com>
Co-authored-by: Reuben Tier <64310361+TheOtterlord@users.noreply.github.com>
Co-authored-by: Yan <61414485+yanthomasdev@users.noreply.github.com>
Co-authored-by: Kevin <46791833+kevinzunigacuellar@users.noreply.github.com>
Co-authored-by: Houston (Bot) <108291165+astrobot-houston@users.noreply.github.com>
Co-authored-by: Junseong Park <39112954+jsparkdev@users.noreply.github.com>
Co-authored-by: liruifengv <25167721+liruifengv@users.noreply.github.com>
Co-authored-by: Yan <61414485+yanthomasdev@users.noreply.github.com>
Co-authored-by: Thomas Bonnet <14293805+thomasbnt@users.noreply.github.com>
Co-authored-by: HiDeoo <494699+HiDeoo@users.noreply.github.com>
Co-authored-by: Houston (Bot) <108291165+astrobot-houston@users.noreply.github.com>
Co-authored-by: Ricardo Jaime Gil Simões <20357414+rgilsimoes@users.noreply.github.com>
Co-authored-by: Yan <61414485+yanthomasdev@users.noreply.github.com>
Co-authored-by: Houston (Bot) <108291165+astrobot-houston@users.noreply.github.com>
Co-authored-by: HiDeoo <494699+HiDeoo@users.noreply.github.com>
Co-authored-by: Sergio A. Arevalo Soria <7903076+sasoria@users.noreply.github.com>
Co-authored-by: Luiz Ferraz <11063910+Fryuni@users.noreply.github.com>
Co-authored-by: Bryce Russell <19967622+BryceRussell@users.noreply.github.com>
Co-authored-by: Atharva <88548999+at-the-vr@users.noreply.github.com>
Co-authored-by: Houston (Bot) <108291165+astrobot-houston@users.noreply.github.com>
Co-authored-by: Sergio A. Arevalo Soria <7903076+sasoria@users.noreply.github.com>
Co-authored-by: HiDeoo <494699+HiDeoo@users.noreply.github.com>
Co-authored-by: Luiz Ferraz <11063910+Fryuni@users.noreply.github.com>
Co-authored-by: Bryce Russell <19967622+BryceRussell@users.noreply.github.com>
Co-authored-by: Reuben Tier <64310361+TheOtterlord@users.noreply.github.com>
Co-authored-by: Houston (Bot) <108291165+astrobot-houston@users.noreply.github.com>
Co-authored-by: Reuben Tier <64310361+TheOtterlord@users.noreply.github.com>
Co-authored-by: HiDeoo <494699+HiDeoo@users.noreply.github.com>
Co-authored-by: Luiz Ferraz <11063910+Fryuni@users.noreply.github.com>
Co-authored-by: Jasper 張 <12832610+jasperteo@users.noreply.github.com>
Co-authored-by: Houston (Bot) <108291165+astrobot-houston@users.noreply.github.com>
Co-authored-by: Reuben Tier <64310361+TheOtterlord@users.noreply.github.com>
Co-authored-by: Atharva <88548999+at-the-vr@users.noreply.github.com>
Co-authored-by: Sarah Rainsberger <5098874+sarah11918@users.noreply.github.com>
Co-authored-by: Lorenzo Lewis <15347255+lorenzolewis@users.noreply.github.com>

**Comment by yanthomasdev** [comment]
Before merging, do not forget:
```
@lunaria-track:src/content/docs/en/community-resources/*.mdx;src/content/docs/en/tutorial/0-introduction/1.mdx;src/i18n/en/nav.ts
```
